.github/pull_request_template.md

```diff
@@ -0,0 +1,33 @@
+<!-- Please see https://github.com/zulip/zulip-terminal#contributor-guidelines ! -->
+
+**What does this PR do?**
+
+Overall description goes here!
+
+- [ ] New Feature?
+- [ ] Fixes Bug? [Fixes <issue>; Partially <issue> | Fixes point <n> of <issue> | Resolves an issue that's not filed]
+- [ ] Internal Refactor?
+
+**Tested?** <!-- Fine to leave some of these unchecked! -->
+- [ ] Manually
+- [ ] Tests adapted (if necessary)
+- [ ] Tests added/changed for any new behavior
+- [ ] Passed linting & tests (overall PR)
+- [ ] Passed linting & tests (each commit)
+
+<!-- See https://github.com/zulip/zulip-terminal#commit-style -->
+**Commit flow** <!-- if more than one commit; add/delete/fill-in as appropriate -->
+* first commit doing some thing
```

Comment:
Although I've used this field in some of my own PR's, I wonder if it's really necessary?
The "Commit flow" is essentially just the commit headers, as we have documented in that link, which can be easily accessed from the commits tab at the top?

Reply:
I agree the commit list can be seen in the PR itself - and does change over time, often, and it's not necessary to keep updating the text here as that happens.

I do feel we could benefit from having something here though - maybe others would like to weigh in.

Reply:
@Rohitth007 That's what the examples are intended to indicate, ie. that it's a summary. I meant to include that they're only ~ commit titles, but I don't want to motivate just copy and pasting those either.